### DISCUSSION TITLE
How to import entire moneydance application data to moneymanagerex?

### DISCUSSION THREAD

**Discussion by devmalick** [moneymanagerex/moneymanagerex#6303] [Q&A]
Hello there i want import entire moneydance data from 2017,  I have tried via QIF importing but showing error and also data are missing, How to resolve the issue please guide?

**Comment by luisdev** [comment]
Providing a bit more detail would be helpful. What data is missing when you import the QIF file into MMX?

Have you checked if that missing data is included in the QIF file that you created? If it's not, then you have  to fix your QIF export process first.

**Reply by devmalick** [reply]
![Screenshot_2023-11-16_08-49-26](https://github.com/moneymanagerex/moneymanagerex/assets/78594080/dd636b0e-eeb4-4ed1-aa7c-56eb1c5a2463)

i have following log during import 
----------------------------------------------------
Line 1 	 !Type:Class
Line 2 	 NMy Finances
Line 3 	 DMy Finances
Line 4 	 ^
Line 5 	 !Type:Cat
Line 6 	 NATM Withdrawal
Line 7 	 DATM Withdrawal
Line 8 	 E
Line 9 	 ^
Line 10 	 NATM Withdrawal:Cash
Line 11 	 DATM Withdrawal:Cash
Line 12 	 E
Line 13 	 ^
Line 14 	 NATM Withdrawal:Service Charge
Line 15 	 DATM Withdrawal:Service Charge
Line 16 	 E
Line 17 	 ^
Line 18 	 NAutomobile
Line 19 	 DAutomobile
Line 20 	 E
Line 21 	 ^
Line 22 	 NAutomobile:Car payment
Line 23 	 DAutomobile:Car payment
Line 24 	 E
Line 25 	 ^
Line 26 	 NAutomobile:Car wash
Line 27 	 DAutomobile:Car wash
Line 28 	 E
Line 29 	 ^
Line 30 	 NAutomobile:Fuel
Line 31 	 DAutomobile:Fuel
Line 32 	 E
Line 33 	 ^
Line 34 	 NAutomobile:Fule
Line 35 	 DAutomobile:Fule
Line 36 	 E
Line 37 	 ^
Line 38 	 NAutomobile:Gasoline
Line 39 	 DAutomobile:Gasoline
Line 40 	 E
Line 41 	 ^
Line 42 	 NAutomobile:Insurance
Line 43 	 DAutomobile:Insurance
Line 44 	 E
Line 45 	 ^
Line 46 	 NAutomobile:Maintenance
Line 47 	 DAutomobile:Maintenance
Line 48 	 E
Line 49 	 ^
Line 50 	 NAutomobile:Parking
-------------------------------------- 8< --------------------------------------
Number of lines read from QIF file: 92432 in 2259 ms
Accounts:
Stock Mandakini
Liablity_m
Receivables_m
Pramod Bk
BF Assets
Liablity_m:Payable_m
Zerodha
Kotak
KCM
Finances
Stock Bfitness
ICICIBank
SagarGems
Assets_m:Apay Wallet
Debendra
INC
Janakalyana Enterprise
Amazon Paylater
Work Account:Biswa Fitness
ICICI Card 3006
TechBix
Wallet BugetBanker
Assets_m
Amazon Pay Card 7009
Work Account
Date Format: 
YYYY-MM-DD
Press OK Button to continue
Line 1 	 !Type:Class
Line 2 	 NMy Finances
Line 3 	 DMy Finances
Line 4 	 ^
Line 5 	 !Type:Cat
Line 6 	 NATM Withdrawal
Line 7 	 DATM Withdrawal
Line 8 	 E
Line 9 	 ^
Line 10 	 NATM Withdrawal:Cash
Line 11 	 DATM Withdrawal:Cash
Line 12 	 E
Line 13 	 ^
Line 14 	 NATM Withdrawal:Service Charge
Line 15 	 DATM Withdrawal:Service Charge
Line 16 	 E
Line 17 	 ^
Line 18 	 NAutomobile
Line 19 	 DAutomobile
Line 20 	 E
Line 21 	 ^
Line 22 	 NAutomobile:Car payment
Line 23 	 DAutomobile:Car payment
Line 24 	 E
Line 25 	 ^
Line 26 	 NAutomobile:Car wash
Line 27 	 DAutomobile:Car wash
Line 28 	 E
Line 29 	 ^
Line 30 	 NAutomobile:Fuel
Line 31 	 DAutomobile:Fuel
Line 32 	 E
Line 33 	 ^
Line 34 	 NAutomobile:Fule
Line 35 	 DAutomobile:Fule
Line 36 	 E
Line 37 	 ^
Line 38 	 NAutomobile:Gasoline
Line 39 	 DAutomobile:Gasoline
Line 40 	 E
Line 41 	 ^
Line 42 	 NAutomobile:Insurance
Line 43 	 DAutomobile:Insurance
Line 44 	 E
Line 45 	 ^
Line 46 	 NAutomobile:Maintenance
Line 47 	 DAutomobile:Maintenance
Line 48 	 E
Line 49 	 ^
Line 50 	 NAutomobile:Parking
-------------------------------------- 8< --------------------------------------
Number of lines read from QIF file: 92432 in 2448 ms
Accounts:
Stock Mandakini
Liablity_m
Receivables_m
Pramod Bk
BF Assets
Liablity_m:Payable_m
Zerodha
Kotak
Assets_m
Amazon Pay Card 7009
Amazon Paylater
ICICI Card 3006
Work Account:Biswa Fitness
Work Account
Wallet BugetBanker
TechBix
SagarGems
KCM
Janakalyana Enterprise
Stock Bfitness
ICICIBank
Finances
Assets_m:Apay Wallet
INC
Debendra
Date Format: 
YYYY-MM-DD
Press OK Button to continue
Date Format: 
DD.MM'YYYY
Date Format: 
DD/MM/YY
Date Format: 
DD MM YY
Date Format: 
DD Mon YY
Date Format: 
DD MM YY

**Comment by vomikan** [comment]
Аdvice: Create one transaction in a new MMEX account and export it to a QIF file.

You will see the structure that MMEX can import. There will be a date, an amount.
This is not in your data. 

This is probably not a complete log and the amounts are there, but the date mask is not correct.
Should be MM/DD/YYYY not DD MM YY

**Reply by luisdev** [reply]
Is the date format not dependant on the Date settings (Regional settings in Windows) on your computer?

**Reply by n-stein** [reply]
> Is the date format not dependant on the Date settings (Regional settings in Windows) on your computer?

This is true for a date picker control such as in the transaction screen, but for QIF import the user has to specify the format of the dates in the upload file so MMEX knows how to convert them to ISO format.

These must match:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/16599929/acd9db84-ebd0-4178-8a8d-247559830b39)
